tests/apps/core/test_authentication_delegation.py

```diff
@@ -47,9 +48,11 @@ def test_user_menu_with_profile_urls(self):
         self.assertIsNotNone(re.search(pattern, str(response.content)))
         self.assertContains(
             response,
-            r'"authentication": {{"endpoint": "{}", "backend": "{}"}}'.format(
-                "https://richie.education:9999",
-                "richie.apps.courses.lms.base.BaseLMSBackend",
+            html.escapejs(
```

Comment:
are we able to foresee what the exact string should be? Because here you are applying the same method as in the template but that does not confirm that the escaping is done as expected and accepting back slashes?

Reply:
Ok I'll explicity write expected output.